packages/module/src/FileDetails/FileDetails.tsx

```diff
@@ -956,7 +956,11 @@ export const FileDetails = ({ fileName }: PropsWithChildren<FileDetailsProps>) =
         <StackItem>
           <span className="pf-chatbot__code-fileName">{path.parse(fileName).name}</span>
         </StackItem>
-        {language && <StackItem className="pf-chatbot__code-language">{language}</StackItem>}
+        {language && (
+          <StackItem data-testid="language" className="pf-chatbot__code-language">
```

Comment:
Could there be more than one of these on a page? Might it be something which is configurable just in case?

Reply:
That's a good callout; I'll work on that.